content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,174 @@
+(new behaviour) rather than to the last one (former behaviour).
+  ([Issue #28269](https://github.com/istio/istio/issues/28269))
+
+## Security
+
+- **Improved** Gateway certificates to be read and distributed from Istiod, rather than in the gateway pods.
+This reduces the permissions required in the gateways, improves performance, and will make certificate reading
+more flexible in the future. This change is fully backwards compatible with the old method, and requires no changes
+to your cluster. If required, it can be disabled by setting the `ISTIOD_ENABLE_SDS_SERVER=false`
+environment variable in Istiod.
+
+- **Improved** TLS configuration on sidecar server side inbound paths to enforce TLSv2 version along with recommended cipher suites.
+             If this is not needed or creates problems with non Envoy clients, it can disabled by setting env variable "PILOT_SIDECAR_ENABLE_INBOUND_TLS_V2" to false.
+
+- **Updated** The ipBlocks/notIpBlocks fields of an AuthorizationPolicy now strictly refer to the source IP address of the IP packet as it arrives to the sidecar.  Prior to this release, if using the Proxy Protocol, then the ipBlocks/notIpBlocks would refer to the IP address determined by the Proxy Protocol.  Now the remoteIpBlocks/notRemoteIpBlocks fields must be used to refer to the client IP address from the Proxy Protocol.
+ ([reference](/docs/reference/config/security/authorization-policy/))([usage](/docs/ops/configuration/traffic-management/network-topologies/))([usage](/docs/tasks/security/authorization/authz-ingress/)) ([Issue #22341](https://github.com/istio/istio/issues/22341))
+
+- **Added** AuthorizationPolicy now supports nested JWT claims.
+  ([Issue #21340](https://github.com/istio/istio/issues/21340))
+
+- **Added** support for client side Envoy secure naming config when trust domain alias is used.
+Fix the multi cluster service discovery client SAN generation: takes all endpoints' service accounts
+into account, rather than the first found service registry.
+
+- **Added** Experimental Feature support allowing Istiod to integrate with external certificate authorities using Kubernetes CSR API (>=1.18 only).
+  ([Issue #27606](https://github.com/istio/istio/issues/27606))
+
+- **Added** Enable user to set the custom vm identity provider for credential authentication
+  ([Issue #27947](https://github.com/istio/istio/issues/27947))
+
+- **Added** action 'AUDIT' to Authorization Policy that can be used to determine which requests should be audited.
+  ([Issue #25591](https://github.com/istio/istio/issues/25591))
+
+- **Added** support for migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
+default, can be disabled by REQUIRE_3P_TOKEN environment variable in Istiod, which will require new tokens with audience. The
+TOKEN_AUDIENCE environment variable allows customizing the checked audience, default remains istio-ca.
```

Comment:
It should be [TOKEN_AUDIENCES](https://github.com/istio/istio/blob/eeeeb3fb10dc77ae85b515e88c7b3b62df563374/pkg/security/security.go#L58)?
```suggestion
`TOKEN_AUDIENCES` environment variable allows customizing the checked audience, default remains istio-ca.
```

Reply:
nice catch